tests/org.jboss.tools.cdi.bot.test/META-INF/MANIFEST.MF

```diff
@@ -14,4 +14,5 @@ Eclipse-BundleShape: jar
 Bundle-Localization: plugin
 Bundle-Vendor: Red Hat
 Import-Package: org.eclipse.jdt.ui.wizards,
- org.jboss.tools.maven.ui.bot.test
+ org.jboss.tools.maven.ui.bot.test,
+ org.jboss.tools.maven.ui.bot.test.utils

```

Comment:
I do no think you can import utils package. Only its parent.

Reply:
Fixed.

Reply:
What does studio say about it? Isn't there bundle xxx cannot be resolved?

Reply:
Fixed. I forget to commit it, I pushed it and after one minute I made force push with fix.

Reply:
Now check are green.